wolox-training/ap-swift
# Book detail

## Summary ##
**Trello card**: https://trello.com/c/MhfItTzu/16-vista-de-detalle-de-un-libro
**Trello card**: https://trello.com/c/0tfL6QLe/18-componente-de-informacion-de-un-libro
**Trello card**: https://trello.com/c/P1fX3yqU/19-tabla-para-comentarios
**Trello card**: https://trello.com/c/H6oPcV8p/21-botones-en-degrade

Book Detail View with book Info cell and comments

## Test Plan ##

  * Open the app
  * Login with Google
  * The main tab bar view should appear after a successfull login.
  * Press over a cell
  * The Book Detail  View view should with book details

## Screenshots ##

### iPhone 7s
![screen shot 2017-03-02 at 9 52 22 am](https://cloud.githubusercontent.com/assets/25587882/23666719/948ba2da-033a-11e7-87d0-8fdc99fe23e7.png)
![libraryview](https://cloud.githubusercontent.com/assets/25587882/23666717/9485335a-033a-11e7-8814-a2d1bc4775be.png)
![screen shot 2017-03-07 at 1 32 03 pm](https://cloud.githubusercontent.com/assets/25587882/23666718/9487219c-033a-11e7-80bc-7de5a62660b7.png)
![screen shot 2017-03-07 at 1 32 20 pm](https://cloud.githubusercontent.com/assets/25587882/23666720/94914bae-033a-11e7-8426-636e694c5e1e.png)

## Screencast ##

![bookdetail](https://cloud.githubusercontent.com/assets/25587882/23666791/bcf19702-033a-11e7-95c2-eb256038c7f8.gif)

The Suggestion bar will is not implemented yet
